### PR TITLE
[WWST-1408] Updates DTH for Kwikset OBSIDIAN Lock

### PR DESCRIPTION
**Pull request by DCzarneckaS**
@greens @MAblewiczS

### CODE REVIEW

File: devicetypes/smartthings/zwave-lock.src/zwave-lock.groovy [hunk 1]
```diff
@@ -531,6 +537,14 @@ private def handleAccessAlarmReport(cmd) {
 			map = [ name: "codeChanged", value: "0 set", descriptionText: "${getStatusForDescription('set')} \"$codeName\"", isStateChange: true ]
 			map.data = [ codeName: codeName, notify: true, notificationText: "${getStatusForDescription('set')} \"$codeName\" in $deviceName at ${location.name}" ]
 			break
+		case 24: //KeyWe manual unlock
+			map = [ name: "lock", value: "unlocked", data: [ method: "manual" ] ]
+			map.descriptionText = "Unlocked manually"
+			break
+		case 25: //Keywe manual lock
```
Comment (mckeed) [inline devicetypes/smartthings/zwave-lock.src/zwave-lock.groovy]: These two values should be in hex (0x18 and 0x19) to match the Z-Wave documentation and surrounding code.

File: devicetypes/smartthings/zwave-lock.src/zwave-lock.groovy [hunk 2]
```diff
@@ -55,6 +55,12 @@ metadata {
 		fingerprint mfr:"0129", prod:"0001", model:"0409", deviceJoinName: "Yale Touchscreen Lever Door Lock" // YRL-220-ZW-605
 		// Samsung
 		fingerprint mfr:"022E", prod:"0001", model:"0001", deviceJoinName: "Samsung Digital Lock" // SHP-DS705, SHP-DHP728, SHP-DHP525
+		//KeyWe
+		fingerprint mfr:"037B", prod:"0002", model:"0001", deviceJoinName: "KeyWe Lock"
+        //KWIKSET OBSIDIAN
+        //zw:Fs type:4003 mfr:0090 prod:0003 model:0742 ver:4.10 zwv:4.34 lib:03 cc:5E,72,5A,98,73,7A sec:86,80,62,63,85,59,71,70,4E,8B,4C,5D role:07 ff:8300 ui:8300
+        fingerprint mfr:"0090", prod:"0003", model:"0742", deviceJoinName: "OBSIDIAN Lock"
```
Comment (mckeed) [inline devicetypes/smartthings/zwave-lock.src/zwave-lock.groovy]: Indentation is wrong here bc it's using spaces, not tabs

**Review by mckeed — CHANGES_REQUESTED** [review]

File: devicetypes/smartthings/zwave-lock.src/zwave-lock.groovy (hunk 2 above)
Comment (greens) [reply devicetypes/smartthings/zwave-lock.src/zwave-lock.groovy]: Also the join name should probably be `Kwikset Obsidian Lock`

File: devicetypes/smartthings/zwave-lock.src/zwave-lock.groovy [hunk 3]
```diff
@@ -14,7 +14,7 @@
  *
  */
 metadata {
-	definition (name: "Z-Wave Lock", namespace: "smartthings", author: "SmartThings", runLocally: true, minHubCoreVersion: '000.017.0012', executeCommandsLocally: false) {
+	definition (name: "Z-Wave Lock", namespace: "smartthings", author: "SmartThings", minHubCoreVersion: '000.017.0012', executeCommandsLocally: false) {
```
Comment (tpmanley) [inline devicetypes/smartthings/zwave-lock.src/zwave-lock.groovy]: The `runLocally` should be left as is

**Comment by DCzarneckaS** [comment]
@greens @MAblewiczS @MMateusiakS  
Please check the latest update.

**Comment by DCzarneckaS** [comment]
Changes are in PR #3395

**Pull request closed without merging**